docs/bridge/bridging/5_return_unused_assets.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Return of unused assets
+sidebar_position: 5
+---
+
+If user locks assets (FTs or NFTs) on the source side connector and there is some kind of problem in destination side or bridge itself
+bridging will not work. If asset is not bridged in the following 4 days from the initial locking transaction asset will no longer be
+eligible for bridging to the destination side.
+
+Currently user with this problem should contact a shard admin, admin should verify that asset can no longer be bridged and return assets to
+the original owner (user). Note that if admin unlocks it before 4 days pass user will effectively get his assets two times since he can
```

Comment:
```suggestion
the original owner (user). Note that if the admin unlocks it before 4 days pass user will effectively get his assets two times since he can
```